### ISSUE TITLE
Comments: keyboard tab highlighting 

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#15940]
- [x] - Tabbing through the comment actions should be an outline instead of a hover state
- [ ] - Add indication that unexpanded comments are highlighted when tabbing through
- [ ] - Pressing enter when a comment is highlighted should expand it
- [ ] - Pressing enter when an action is highlighted should perform the action.

**Comment by gwwar** [comment]
Thanks @ryelle! I think `getKeyboardHandler` would be useful enough to move up to the top lib dir. We may want to choose a different name to better describe what it does. Perhaps something like `accessibilityKeyHandler`? I'm terrible at naming things, but I'm sure we can come up with something we both like.

**Comment by ryelle** [comment]
Ha, it's already been through the rename cycle once :) I'm hesitant to call it `accessible*` because I don't want someone to think that's all they need to do to make an element accessible — it also needs to be focusable (tabindex=0), and needs to have design for that focus state (and for the best accessibility, there should probably be thought about why X element can't just be a button/link).

That said, maybe it's not an issue, and I can't think of another name... so maybe `accessibleKeyHandler` works.
